tools/java/org/unicode/cldr/util/DtdData.java

```diff
@@ -224,7 +224,10 @@ public void addComment(String commentIn) {
                     attributeStatus = AttributeStatus.value;
                     break;
                 case "@DEPRECATED":
-                    isDeprecatedAttribute = true;
+                    if (!name.equals("draft")
```

Comment:
For the others ("collation", "transform", "unitPreferenceData") I think we explicitly removed the @Deprecated on the element, which is cleaner than having this special test. 

(There is no particular value to distinguishing the "with-children" case from the "without children" case.)